libs/langchain/langchain/chains/combine_documents/stuff.py

```diff
@@ -1,19 +1,40 @@
 """Chain that combines documents by stuffing into context."""
-
 from typing import Any, Dict, List, Optional, Tuple
 
 from langchain_core.documents import Document
 from langchain_core.prompts import BasePromptTemplate, format_document
 from langchain_core.prompts.prompt import PromptTemplate
 from langchain_core.pydantic_v1 import Extra, Field, root_validator
+from langchain_core.runnables import Runnable, RunnableParallel
 
 from langchain.callbacks.manager import Callbacks
-from langchain.chains.combine_documents.base import (
-    BaseCombineDocumentsChain,
-)
+from langchain.chains.combine_documents.base import BaseCombineDocumentsChain
 from langchain.chains.llm import LLMChain
 
 
+def create_stuff_documents_chain(
+    llm_chain: Runnable,
+    document_input_key: str,
```

Comment:
I think people get confused by this `input_key` `output_key` pattern quite often - could we just take a list of documents?